egs2/TEMPLATE/tts1/README.md

```diff
@@ -317,6 +317,11 @@ You can change via `--g2p` option in `tts.sh`.
 - `espeak_ng_russian`: [espeak-ng/espeak-ng](https://github.com/espeak-ng/espeak-ng)
     - e.g. `Привет мир.` -> `[p, rʲ, i, vʲ, ˈe, t, mʲ, ˈi, r, .]`
     - This result provided by the wrapper library [bootphon/phonemizer](https://github.com/bootphon/phonemizer)
+- `g2p_en`: [Kyubyong/g2pK](https://github.com/Kyubyong/g2pK)
```

Comment:
maybe g2p_en -> g2pk (and also the next item is g2pk_no_space)?

Reply:
Thanks! It is typo.